### PR TITLE
BaSD-MS-31: Admin redux

### PR DESCRIPTION
**Pull request by GuidoGlielmi**

### CODE REVIEW

**Comment by nicolasblasco** [comment]
@GuidoGlielmi we should keep using the React States to handle the input values.

**Comment by nicolasblasco** [comment]
@GuidoGlielmi we should keep using the React States to handle the input values. So SET_ADMIN and CLEAR_ADMIN isn't needed.

**Comment by nicolasblasco** [comment]
@GuidoGlielmi we should keep using the React States to handle the input values. So SET_ADMIN and CLEAR_ADMIN isn't needed.

File: src/redux/Admins/reducer.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  SET_ADMIN,
+  CLEAR_ADMIN,
+  ADMIN_CLOSE_ERROR_MODAL
+} from '../../constants/actionTypes';
+
+const initialState = {
+  isLoading: false,
+  admins: [],
+  admin: {},
+  error: { show: false }
```
Comment (nicolasblasco) [inline src/redux/Admins/reducer.js:26]: @GuidoGlielmi maybe we should also add message and title to the initialState of error. What do you think?

File: src/redux/Admins/thunks.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+      .then((response) => {
+        if (response.status !== 200 && response.status !== 201 && response.status !== 204) {
+          const status = `${response.status} ${response.statusText}`;
+          return response.json().then(({ message }) => {
+            if (message.message) throw { message: message.message, status };
+            throw { message, status };
+          });
+        }
+        dispatch(updateAdminFulfilled());
+        window.location.href = '/admins';
```
Comment (nicolasblasco) [inline src/redux/Admins/thunks.js:117]: @GuidoGlielmi maybe we should implement useHistory. What do you think?

**Comment by nicolasblasco** [comment]
@GuidoGlielmi Very good PR! It works perfectly! I left some comments with some changes to be made and some suggestions that we could debate.

**Review by PabloMBernal — APPROVED** [review]

**Review by nicolasblasco — APPROVED** [review]

**Pull request merged**